packages/wouter/src/use-browser-location.js

```diff
@@ -40,8 +40,11 @@ export const usePathname = ({ ssrPath } = {}) =>
     ssrPath ? () => ssrPath : currentPathname
   );
 
-export const navigate = (to, { replace = false } = {}) =>
-  history[replace ? eventReplaceState : eventPushState](null, "", to);
+const currentHistoryState = () => history.state;
+export const useHistoryState = () => useLocationProperty(currentHistoryState);
```

Comment:
We can safely assume that `state` in SSR is always `null`, so let's provide a second argument here.